CHANGELOG.md

```diff
@@ -3,22 +3,24 @@ All notable changes to this project will be documented in this file. This change
 
 ## [Unreleased]
 ### Changed
-- Add a new arity to `make-widget-async` to provide a different widget shape.
 
-## [0.1.1] - 2017-05-08
-### Changed
-- Documentation on how to make the widgets.
+## [1.1.1]
```

Comment:
Not sure if we want to continue with dates at all, but if we do, these last couple are missing them.